doc/source/conf.py

```diff
@@ -108,6 +108,8 @@
     "transformers",
     "tqdm",
     "dill",
+    "pydantic",
```

Comment:
After the docstrings are rendering, would make sure that mocking `pydantic` doesn't result in weird things, i.e. would be nice to have readable `pydantic` models in the API docs.

Reply:
This is resolved implicitly since we are now using `sphinx-autoapi` (https://github.com/SeldonIO/alibi-detect/pull/482), meaning modules do not require mocking.